src/js/models.js

```diff
@@ -123,40 +132,61 @@ Models.Room = BaseModel.extend({
         app.get('messages').parseModel(app, data.messages[idx]);
       }
     });
+  },
+  sendText: function(text) {
+    this.get('app').get('websocket').sendCommand('account.account_service.room.message.post', {
+      'text': text,
+      'account_id': this.get('account').get('id'),
+      'account_service_id': this.get('connectors').findWhere({'serviceType': 'session'}).get('id'),
+      'type': 'message',
+      'room_id': this.get('id')
+    });
   }
 }, {
   parseModel: function (app, model) {
     model.account = app.get('accounts').parseModel(app, model.accountSnapshot);
     delete model.accountSnapshot;
 
-    model.connectors = model.accountServicesSnapshots.map(function(item) {
+    model.connectors = new Models.ConnectorCollection();
+    model.connectors.add(model.accountServicesSnapshots.map(function(item) {
       return app.get('connectors').parseModel(app, item);
-    });
+    }));
     delete model.accountServicesSnapshots;
 
     let peers = new Models.PeerCollection();
     for (let idx in model.peers) {
       peers.parseModel(app, model.peers[idx]);
     }
     model.peers = peers;
+    model.app = app;
+
+    // Workaround: datetime must come as timestamps.
+    let d = model.lastMessageTimestamp.split(" ");
+    d[0] = d[0].split('-').reverse().join('-');
+    d = Date.parse(d.join('T'));
+    model.sorting = -d;
 
     return new this(model);
   }
 });
 
 Models.RoomCollection = BaseModule.extend({
   model: Models.Room,
+  comparator: 'sorting',
+
   dispatch: function (app, method, params) {
     switch(method) {
         case 'message.received':
           app.get('messages').parseModel(app, params.message);
           break;
+        default:
+          console.log('Unknown methood: ' + method);
     }
   },
   start: function (app) {
     var self = this;
     $.ajax({
-      url: 'https://' + app.get('backendUrl') + '/rest/room?limit=20',
+      url: 'https://' + app.get('backendUrl') + '/rest/room',
```

Comment:
why removing the limit 20 here? The first time we call it, will it try to load all the old rooms of the user?

Reply:
It's no necessary now. It can manage all rooms.